### ISSUE TITLE
Accessibility: Buttons in event list widget are not labeled

### ISSUE THREAD

**Issue by heull001** [FossifyOrg/Calendar#502]
### Checklist

- [x] I can reproduce the bug with the latest version given [here](https://github.com/FossifyOrg/Calendar/releases/latest).
- [x] I made sure that there are **no existing issues** - [open](https://github.com/FossifyOrg/Calendar/issues) or [closed](https://github.com/FossifyOrg/Calendar/issues?q=is%3Aissue+is%3Aclosed) - to which I could contribute my information.
- [x] I made sure that there are **no existing discussions** - [open](https://github.com/FossifyOrg/Calendar/discussions) or [closed](https://github.com/FossifyOrg/Calendar/discussions?discussions_q=is%3Aclosed) - to which I could contribute my information.
- [x] I have read the FAQs inside the app (Menu -> About -> FAQs) and my problem isn't listed.
- [x] **I have taken the time to fill in all the required details. I understand that the bug report will be dismissed otherwise.**
- [x] This issue contains only one bug.
- [x] I have read and understood the [contribution guidelines](https://github.com/FossifyOrg/Calendar/blob/master/CONTRIBUTING.md).

### Affected app version

1.2.0

### Affected Android/Custom ROM version

GrapheneOS 2025030300

### Affected device model

Pixel 8a

### How did you install the app?

F-Droid / IzzyOnDroid

### Which calendar service or provider do you use, if any?

Radicale

### Which sync adapter app do you use for calendar synchronization, if any?

DAVx5

### Steps to reproduce the bug

- add event list widget to home screen
- enable talkback (<https://support.google.com/accessibility/android/answer/6007100?hl=en>)
- Focus the widget and swipe right or left to the buttons (the right is for adding an event, the left I don't know)

### Expected behavior

Talkback should say the function of the buttons

### Actual behavior

Talkback says "unnamed".

**Comment by naveensingh** [comment]
> the left I don't know

It's the "Go to today" button :)

**Comment by naveensingh** [comment]
It'll be fixed as part of #5
